guides/v2.2/extension-dev-guide/validate/test-module.md

```diff
@@ -1,58 +1,71 @@
 ---
 group: php-developer-guide
-subgroup: 06_Validate
 title: Test your component
-menu_title: Test your component
-menu_order: 2
-menu_node:
 redirect_from:
-  - /guides/v2.0/mktpl-quickstart/dev-test.html
-  - /guides/v2.1/mktpl-quickstart/dev-test.html
   - /guides/v2.2/mktpl-quickstart/dev-test.html
 ---
 
 ## Unit and Integration Tests {#test-unit}
 
 Run the PHPUnit based Magento unit and integration tests.
-For more information see the [Magento Testing Overview]({{ page.baseurl }}/test/testing.html)
+See the [Magento Testing Overview].
 
 ## Functional testing {#test-functional}
 
 For further testing with the Magento functional testing frameworks, see
-[Functional Testing Framework]({{ page.baseurl }}/mtf/mtf_introduction.html).
+[Functional Testing Framework] and [Magento Functional Testing Framework].
 
 ## Test using {{site.data.var.ce}} {#test-comm}
 
-Test your component by deploying {{site.data.var.ce}} and adding the component to the project's <code>composer.json</code>.
+Test your component by deploying {{site.data.var.ce}} and adding the component to the project's `composer.json`. To install, see [Install using Composer].
 
-{% highlight JSON %}
+```json
 "require": {
     "magento/magento-composer-installer": "*",
-    "magento/product-community-edition": "2.0.0",
     "yourvendorname/module-one": "0.1.1"
 },
-{% endhighlight %}
+```
 
-Remember to [register]({{ page.baseurl }}/extension-dev-guide/build/component-registration.html) your component as well, adding the location of your component. Confirm that your component works as expected and Magento functionality is not compromised.
+Remember to [register] your component as well, adding the location of your component. Confirm that your component works as expected and Magento functionality is not compromised.
 
 ## Test installing your component {#test-install}
 
-Before you publish your component, you should test installing it using the [Magento Component Manager]({{ page.baseurl }}/comp-mgr/bk-compman-upgrade-guide.html){: target="_blank"} (part of the Magento Admin).
+Before you publish your component, you should test installing it using the [Magento Component Manager]. Access this feature through the part of the [Web Setup Wizard] in the Magento Admin.
```

Comment:
you need to test the component installation using .....

"the part of the [Web Setup Wizard] in the Magento Admin."

What is "the part"?   (Again I don't have context, but it sounds strange -- does it mean -- Access this feature through the Web Setup Wizard available in the Magento Admin?)